.travis.yml

```diff
@@ -15,12 +13,17 @@ matrix:
         # Lint the first commit in the PR.
         - \[ -z "$TRAVIS_COMMIT_RANGE" \] || (echo -e '\nLinting the commit message according to the guidelines at https://goo.gl/p2fr5Q\n' && git log $TRAVIS_COMMIT_RANGE --pretty=format:'%h' --no-merges | tail -1 | xargs npx -q core-validate-commit --no-validate-metadata)
     - name: "Test Suite"
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - g++-4.9
       install:
+        - export CC='ccache gcc-4.9' CXX='ccache g++-4.9'
         - ./configure
-        - make -j2 V=
+        - make -j2 V=1 | sed -e "s/'.*\$//" -e "s|$PWD||g"
```

Comment:
It seems https://github.com/nodejs/build/issues/1517 is still being discussed. I'm not a fan of this change, can we revert it?

Reply:
ack